app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,29 @@
  *= require_tree .
  *= require_self
  */
+
+ <style>
+a:link {
+  color: green;
+  background-color: transparent;
+  text-decoration: none;
+}
+
+a:visited {
+  color: purple;
+  background-color: transparent;
+  text-decoration: none;
+}
+
+a:hover {
+  color: red;
+  background-color: transparent;
+  text-decoration: underline;
+}
+
+a:active {
+  color: yellow;
+  background-color: transparent;
+  text-decoration: underline;
+}
```

Comment:
where does this CSS actually implied ?
Can you add a screenshot showing the result of this CSS ?

Reply:
1.Here
![Screenshot from 2022-12-21 18-34-52](https://user-images.githubusercontent.com/115778571/208912148-0fcce437-1083-450b-9ddb-971e6c6d751f.png)